Projects/Feature/Chat/Sources/CreateCheckList/CreateCheckListView.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  CreateCheckListView.swift
+//  FeatureChat
+//
+//  Created by 현수빈 on 8/28/24.
+//
+
+import SwiftUI
+
+import CoreDomain
+
+import ComposableArchitecture
+import SharedDesignSystem
```

Comment:
```suggestion
import SwiftUI

import CoreDomain
import SharedDesignSystem

import ComposableArchitecture
```

SharedDesignSystem도 위로 옮겨주세요!